docs/features/setting-storage.md

```diff
@@ -0,0 +1,116 @@
+# Setting Storage Extensions
+
+:heavy_exclamation_mark: There is also a version of [this document with code samples in VB.Net](./setting-storage.vb.md) :heavy_exclamation_mark: |
+----------------------------------------------------------------------------------------------------------------------------------------------- |
+
+SettingsStorageExtensions is a helper class that groups extension methods to facilitate the use of serialization, handling and storing of any content in a UWP application. This extension wraps the handling and storing of content (for example: custom content, key-values, files, etc.) in the same way as simple settings for a consistent interface.
+
+## Advantages:
+ - All methods of handling and storing content can be found in the same class and using a similar nomenclature. 
+ - Facilitate the serialization and deserialization of data content and make it transparent for the developers.
+ - Using methods that avoid problems in the serialization process (for example: big files that can take a long time to process).
+ - All methods automatically give default values back if the key you're trying to read doesn't exist.
+ - It makes working with byte arrays as easy as working with non-binary data (which can sometimes be complicated.)
+
+This helper contains extension methods for handling and storing content in:
+ - StorageFolder
+ - ApplicationData
+
+More info: [https://docs.microsoft.com/es-es/windows/uwp/design/app-settings/store-and-retrieve-app-data](https://docs.microsoft.com/es-es/windows/uwp/design/app-settings/store-and-retrieve-app-data).
```

Comment:
For additional reference, it would be good to mention the `StorageFileHelper` in the toolkit and highlight how these are different and complimentary.

Reply:
I´m not sure if this adds value to our documentation. It could cause confusion, as this helper is not a part of our generated code.